css/style.css

```diff
@@ -0,0 +1,34 @@
+header {
+    position: fixed;
+    width: 100%;
+  }
+
+body{
+  display:flex;
+  flex-direction:column;
+  background-color:rgb(222, 232, 203);
+  text-align:center;
+  font-family:verdana;
+}
+
+#header-img{
+  width: 100px;
+  text-align:center;
+}
+
+#video{

```

Comment:
- [ ]  I think you should add a white space after `#video` to keep CSS best practices 🤓